apps/main/src/dao/components/PageGauges/GaugeVoting/index.tsx

```diff
@@ -2,19 +2,17 @@ import { useEffect } from 'react'
 import styled from 'styled-components'
 import { WrongNetwork } from '@/dao/components/PageVeCrv/WrongNetwork'
 import useStore from '@/dao/store/useStore'
-import { isLoading } from '@ui/utils'
-import { ConnectWalletPrompt, useWallet } from '@ui-kit/features/connect-wallet'
-import { useApiStore } from '@ui-kit/shared/useApiStore'
+import type { CurveApi } from '@/dao/types/dao.types'
+import { ConnectWalletPrompt, isLoading, useConnection, useWallet } from '@ui-kit/features/connect-wallet'
 import CurrentVotes from './CurrentVotes'
 
 const GaugeVoting = ({ userAddress }: { userAddress: string | undefined }) => {
   const getUserGaugeVoteWeights = useStore((state) => state.user.getUserGaugeVoteWeights)
   const userGaugeVoteWeightsMapper = useStore((state) => state.user.userGaugeVoteWeightsMapper)
-  const curve = useApiStore((state) => state.curve)
+  const { lib: curve } = useConnection<CurveApi>()
```

Comment:
```suggestion
  const { connectState, lib: curve } = useConnection<CurveApi>()
```

Then remove `const { connectState } = useConnection()`